### PR TITLE
Generic App Share to Stage Capability: Fixes and UI Screenshots update

### PR DESCRIPTION
**Pull request by Keshav-MSFT**

### CODE REVIEW

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
     "@testing-library/user-event": "^13.5.0",
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json:12]: Why did you removed package-lock.json ??

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
     "fluid-framework": "~0.59.0",
     "jquery": "^3.6.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "17.0.0",
+    "react-dom": "17.0.0",
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json:16]: Why did you downgraded the packages ?
Ideal way is to use latest versions.

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json [hunk 3]
```diff
@@ -12,10 +12,10 @@
...
     "react-router-dom": "6.2.2",
-    "react-scripts": "^4.0.0",
+    "react-scripts": "4.0.0",
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json:18]: Why did you remove ^ from the package ?

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js [hunk 4]
```diff
@@ -1,11 +1,10 @@
...
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import { AppRoute } from './router/router';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <AppRoute />
-  </React.StrictMode>
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:10]: Why did we remove these code ?

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js [hunk 5]
```diff
@@ -1,11 +1,10 @@
...
-);
+ReactDOM.render(
+    <React.StrictMode>
+        <AppRoute />
+    </React.StrictMode>, document.getElementById('root')
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:9]: Why did you remove configureController.cs file ?

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js [hunk 6]
```diff
@@ -1,11 +1,10 @@
...
 import { AppRoute } from './router/router';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <AppRoute />
-  </React.StrictMode>
-);
+ReactDOM.render(
+    <React.StrictMode>
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:7]: image

Image name is wrong:
"screen" but not 3 time e

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js [hunk 7]
```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import { AppRoute } from './router/router';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:6]: Can we fix the alignment for image ?
image

Comment (ChetanSharma-msft) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:6]: Also, we could have provided proper app name
image

File: samples/meetings-stage-view/nodejs/ClientApp/src/index.js [hunk 8]
```diff
@@ -1,11 +1,10 @@
...
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <AppRoute />
-  </React.StrictMode>
-);
+ReactDOM.render(
+    <React.StrictMode>
+        <AppRoute />
+    </React.StrictMode>, document.getElementById('root')
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/ClientApp/src/index.js:9]: How this change fixing our issues ?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Reviewed and provided my comments

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json (hunk 1 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json:12]: Restored

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json (hunk 2 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json]: Resolved dependency with other package and restored latest version

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js (hunk 4 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:10]: Restored as per React 18.2.0 recommendation.

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js (hunk 5 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js]: Migration to React doesn't required controller files

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js (hunk 6 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:8]: Removed additional unnecessary images and updated file names

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js (hunk 7 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js:6]: Fixed alignment by moving inputs to next line

File: samples/meetings-stage-view/nodejs/ClientApp/src/index.js (hunk 8 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/nodejs/ClientApp/src/index.js]: Reverted

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]
